tests/define-arrow.test.js

```diff
@@ -0,0 +1,342 @@
+'use strict';
+
+const { AMD_DEFINE_RESULT } = require('../src/constants');
+
+describe('Plugin for define blocks', () => {
```

Comment:
Let's update the suite description to "Plugin for define blocks with arrow function factories"